share/keepassxc.ini

```diff
@@ -17,6 +17,7 @@ LastOpenedDatabases=@Invalid()
 [GUI]
 Language=system
 ShowTrayIcon=false
+DarkTrayIcon=false
```

Comment:
Could you please add a default for this config option in Config::init() (src/core/Config.cpp) as well? Rest looks good, thanks!